pylint/test/functional/ternary.txt

```diff
@@ -5,3 +5,4 @@ consider-using-ternary:21:func3:Consider using ternary (true_value if condition
 consider-using-ternary:24::Consider using ternary ('ERROR' if some_callable(condition) else 'SUCCESS'):HIGH
 consider-using-ternary:25::Consider using ternary ('greater' if SOME_VALUE1 > 3 else 'not greater'):HIGH
 consider-using-ternary:26::Consider using ternary ('both' if (SOME_VALUE2 > 4) and (SOME_VALUE3) else 'not'):HIGH
+consider-using-ternary:29::Consider using ternary (SOME_VALUE2):HIGH
```

Comment:
How about "Ternary can be simplified (SOME_VALUE2)", "Ternary can be replaced with (SOME_VALUE2)"?